x-pack/plugins/fleet/common/constants/output.ts

```diff
@@ -29,6 +29,7 @@ export const DEFAULT_OUTPUT: NewOutput = {
 export const SERVERLESS_DEFAULT_OUTPUT_ID = 'es-default-output';
 
 export const LICENCE_FOR_PER_POLICY_OUTPUT = 'platinum';
+export const LICENCE_FOR_OUTPUT_PER_INTEGRATION = 'enterprise';
```

Comment:
@nimarezainia @jen-huang It is not weird to have a different licence for that feature that the per policy output? it seems to be with that feature a user will be able to achieve almost the same result if it set an output for all the integration of that policy

Reply:
@nchaulet enterprise is higher than platinum, so it is not possible for a user to have access to output per integration but not per policy output